clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/ClinvarPipeline.scala

```diff
@@ -47,6 +47,8 @@ object ClinvarPipeline {
     val scvObservations = mapper.mapScvObservations(archiveBranches.scvObservations)
     val scvTraitSets = mapper.mapScvTraitSets(archiveBranches.scvTraitSets)
     val scvTraits = mapper.mapScvTraits(archiveBranches.scvTraits)
+    val vaTraitSets = mapper.mapVaTraitSets(archiveBranches.vaTraitSets)
```

Comment:
In a past discussion we'd said that it'd be good to have trait and trait-set be conceptually distinct from Clinvar-specific artifacts, so:
```suggestion
    val traitSets = mapper.mapVaTraitSets(archiveBranches.vaTraitSets)
```